Api/Resource/Front/UserFavouritesDeleteResource.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the DreamCommerce Shop AppStore package.
+ *
+ * (c) DreamCommerce
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace DreamCommerce\Component\ShopAppstore\Api\Resource\Front;
+
+class UserFavouritesDeleteResource extends UserFavouritesResource
+{
+    public function getMethod(): string
+    {
+        return 'DELETE';
+    }
```

Comment:
usuwanie z ulubionych chyba wymaga zalogowania, nie?
Zobacz tutaj się odbywa logowanie na podstawie metodki isPublic()
https://github.com/dreamcommerce/shop-appstore-lib/blob/dev2.0/Api/Resource/Front/FrontResource.php#L94

Reply:
hmm, weryfikację czy user zalogowany jest w https://github.com/dreamcommerce/shoper/pull/5637/files Webapi_Front_Module_User_Favourites bo jednak testy E2E (do których wykorzystywana jest ta paczka ) to case jeden - próbuję zrozumieć w jakim celu tu jeszcze raz się odbywa weryfikacja czy zalogowany jak robi to końcówka API? :thinking:

Reply:
I też ciut inaczej pytanie, w jakim celu tutaj odbywa się za każdym razem request aby zalogować jak dam to isPublic na false? :thinking: